src/Sylius/Bundle/AttributeBundle/Resources/public/js/sylius-attribute.js

```diff
@@ -0,0 +1,99 @@
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+(function ($) {
+    'use strict';
+
+    function calculateNextIndex() {
+        var nextIndex = 0;
+        var attributeItems = $('#attributes > div .collection-item');
```

Comment:
Totally valid issue but we won't have this case so I am fine with leaving it as is and adding a new issue after this is merged to keep track of this problem.